lib/Parse/ParseDecl.cpp

```diff
@@ -574,6 +574,7 @@ ParserResult<AvailableAttr> Parser::parseExtendedAvailabilitySpecList(
     }
     assert(PlatformAgnostic == PlatformAgnosticAvailabilityKind::None);
 
+    // ALLANXXX move this up
```

Comment:
Leftover comment?

Reply:
Ugh yes, thanks

Reply:
Fixed. I need a push hook that's reminds me about leaving TODOs in like that.